code/modules/shuttle/computers/escape_pod_computer.dm

```diff
@@ -66,6 +66,9 @@
 	var/obj/docking_port/mobile/crashable/escape_shuttle/shuttle = SSshuttle.getShuttle(shuttleId)
 	switch(action)
 		if("force_launch")
+			if(!pod_state == STATE_READY && !pod_state == STATE_DELAYED)
```

Comment:
shouldn't this be
if(pod_state != STATE_READY && pod_state != STATE_DELAYED)

Reply:
Yes